docs/nd4j/templates/quickstart.md

```diff
@@ -0,0 +1,381 @@
+
+To follow the examples in this quick start you will need to know some Java. You will also need to install the following software on your computer:
+<!--- from the dl4j quickstart, pointing to the dl4j quiclstart for details. -->
+* [Java (developer version)](./deeplearning4j-quickstart#Java) 1.7 or later (Only 64-Bit versions supported)
+* [Apache Maven](./deeplearning4j-quickstart#Maven) (automated build and dependency manager)
+<!--- git allows us to start with a cleaner project than mvn create. -->
+* [Git](./deeplearning4j-quickstart#Git)(distributed version control system)
+
+If you are confident you know how to use maven and git, please feel free to skip to the [Basics](#Basics). In the remainder of this section we will build a small 'hello ND4J' application to verify the prequisites are set up correctly.
+
+Execute the following commands to get the project from github. 
+
+<!--- TODO: Create HelloNd4J or Quickstart-nd4j repo in Deeplearning4J. -->
+```shell
+git clone https://github.com/RobAltena/HelloNd4J.git
+
+cd HelloNd4J
+
+mvn install
+
+mvn exec:java -Dexec.mainClass="HelloNd4j"
+```
+
+When everything is set up correctly you should see the following output:
+
+```shell
+SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
+SLF4J: Defaulting to no-operation (NOP) logger implementation
+SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
+[         0,         0]
+```
+
+## Basics
+<!--- TODO: We will put some into this page. Start with refering to existing doc. -->
+While this quickstart is being build, please refer to our existing 
+[basics usage](./nd4j-basics) document.
+
+The main feature of Nd4j is the versatile n-dimensional array interface called INDArray. To improve performance Nd4j uses off-heap memory to store data. The INDArray is different from standard Java arrays.
+
+Some of the key properties and methods for an INDArray x are as follows:
+
+```java
+// The number of axes (dimensions) of the array.
+int dimensions = x.shape().length;
+
+// The dimensions of the array. The size in each dimension.
+long[] shape = x.shape();
+
+// The total number of elements.
+length = x.length();
+
+// The type of the array elements. 
+DataType dt = x.dataType();
+```
+<!--- staying away from itemsize and data buffer. The numpy quickstart has these. -->
+
+### Array Creation
+To create INDArrays you use the static factory methods of the [Nd4j](https://deeplearning4j.org/api/latest/org/nd4j/linalg/factory/Nd4j.html ) class.
+
+<!--- We have good docs on creating INDArrays already.  -->
+<!--- https://deeplearning4j.org/docs/latest/nd4j-overview#creating -->
+The `Nd4j.create` function is overloaded to make it easy to create INDArrays from regular Java arrays. The example below uses Java `double` arrays. Similar create methods are overloaded for `float`, `int` and `long`.
+
+```java
+double arr_2d[][]={{1.0,2.0,3.0},{4.0,5.0,6.0},{7.0,8.0,9.0}};
+INDArray x_2d = Nd4j.create(arr_2d);
```

Comment:
Let's direct users to Nd4j.createFromArray instead.
Unlike Nd4j.create, there's overloads for all java primitive types and Number (long, short, Long, Boolean etc), and so users are less likely to get confused with Nd4j.create(int[]), where they think the int[] is content just like Nd4j.create(double[]), but it's actually shape